config.py

```diff
@@ -1250,6 +1270,19 @@
     ```
 """
 
+n_freqs: int = 5
+"""
+The number of frequencies used for decoding using time frequency widows.
```

Comment:
```suggestion
The number of frequencies used for decoding using time frequency windows.
```
is this really needed?

Reply:
Yes I think it's needed. The other possibility would be to specify n_freq_windows.
(n_freq_windows = n_freqs - 1)

Reply:
I don't get this parametrization. I would say you need a list of tuples with (freq_min, freq_max) for every fred band or allow an array and decode between each value. clear?

Reply:
Solved in the last commit